notebooks/py_scripts/2022-allen-institute-workshop.py

```diff
@@ -0,0 +1,127 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.14.0
+#   kernelspec:
+#     display_name: Python 3.7.9 ('workflow-calcium-imaging')
+#     language: python
+#     name: python3
+# ---
+
+# # Allen Institute Ephys Workshop
+# September 22, 2022
+# + In this notebook, we will show how to interact with a database in Python and how export data into a Neurodata Without Borders (NWB) file.
+#
+# + Other notebooks in this directory describe the process for running the analysis steps in more detail.
+#
+# + This notebook is meant to be run on CodeBook (`https://codebook.datajoint.io`) which contains example data.
+#
+# + First run the `01-configure` and `04-automate` notebooks to set up your environment and load example data into the database, respectively.
+
+# ## Configuration
+
+import datajoint as dj
+import numpy as np
+from matplotlib import pyplot
+
+# Enter database credentials.  A DataJoint workflow requires a connection to an existing relational database. The connection setup parameters are defined in the `dj.config` python dictionary.
+
+# + tags=[]
+dj.config['custom'] = {'database.prefix': '<username>_allen_ephys_',
+                       'ephys_root_data_dir': ["/tmp/test_data/workflow_ephys_data1/",
+                                            "/tmp/test_data/workflow_ephys_data2/",
+                                            "/tmp/test_data/workflow_localization/", 
+                                            "/home/inbox/0.1.0a4/workflow_ephys_data1/",
```

Comment:
`tmp` is just for local download. if uniquely on codebook, not needed

```suggestion
                       'ephys_root_data_dir': [
                                            "/home/inbox/0.1.0a4/workflow_ephys_data1/",
```

Reply:
Good catch